src/Altinn.Platform/Altinn.Platform.Storage/Storage/Repository/InstanceRepository.cs

```diff
@@ -114,7 +115,7 @@ public async Task<bool> Delete(Instance item)
                     ResponseContinuationTokenLimitInKb = 7
                 };
 
-                if (continuationToken != null)
+                if (continuationToken != null && !string.IsNullOrEmpty(continuationToken))
```

Comment:
Checking for null twice

Reply:
Det first check doesn't cover the empty string, so will remove that